### PR TITLE
Add test for error logging

### PR DESCRIPTION
**Pull request by deejay1**
Also drop mock test dependency as it's included in Python>=3.3

### CODE REVIEW

**Comment by pnuckowski** [comment]
:+1:

File: tests/test_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from unittest import TestCase
+
+from alamo_scheduler.scheduler import AlamoScheduler
```
Comment (pnuckowski) [inline tests/test_scheduler.py]: order

Comment (deejay1) [inline tests/test_scheduler.py]: fixed

**Comment by h00dy** [comment]
:yum:

**Pull request merged**